README.md

```diff
@@ -77,8 +78,10 @@ On default, make install puts the compiled binary in `go/bin`.
   - `depesz` - https://explain.depesz.com [default]
   - `dalibo` - https://explain.dalibo.com
   - `tensor` - https://explain.tensor.ru
+  - `custom` - Any visualizer that you wish to use.  If using a `custom` target, you will need to provdie the following:
+    - `--post_url` - The absolute URL to which the `<form>` will `POST` to.  A good reference would be to look for the `action` param in the `<form>` tag.  Note that `plan-exporter` only works with visualizers supporting HTTP `POST` at this time.
+    - `--plan_key` - The `name` of the `<textarea>` (or other input) element into which the query plan gets pasted
```

Comment:
I suggest adding only the `post_url` parameter. In my opinion, it would be better to explicitly implement the new type of goal, rather than modify the `plan_key` option.

Reply:
I agree, but how do you plan to get the `plan_key`? Would we write an HTML scraper to fetch the HTML element, or something like that (or maybe there's something more elegant that I'm not seeing)?

Reply:
We can simply add a new package to the visualizer, if necessary.
Unlike `post_url`, there are few visualization services, and they can be explicitly added as code.
However, `post_url` should be customizable.

Reply:
Agreed -- there are not that many visualizer services out there, so it's not going to be too much trouble to create new packages as needed.  In that case, it may make more sense to abandon this PR and add packages via PR as needed.

Or were you thinking that someone could say `--target=depesz --post_url=x.x.x.x`, and then the `depesz.go` package would use the provided `post_url` instead of the default `http://explain.depesz.com/plan`?  If that's case, I can roll most of these changes back and just update to handle a custom `--post_url`

Reply:
Exactly, it would be cool to use a custom `post_url`

Reply:
I have added the post_url arg as an option